js_modules/dagit/packages/core/src/app/Flags.tsx

```diff
@@ -10,6 +10,7 @@ export enum FeatureFlag {
   flagDisableWebsockets = 'flagDisableWebsockets',
   flagNewWorkspace = 'flagNewWorkspace',
   flagRunBucketing = 'flagRunBucketing',
+  flagAssetGraphExperiments = 'flagAssetGraphExperiments',
```

Comment:
Intentionally a generic name related to any asset graph experiments? Or should this be more specifically named in reference to the zoom level changes?

Reply:
Ahh good call I'll change this, I think this came from the old diff I restored to get some of this work!